src/control/cmd/daos/filesystem.go

```diff
@@ -44,9 +44,10 @@ type fsCmd struct {
 type fsCopyCmd struct {
 	daosCmd
 
-	Source   string `long:"src" short:"s" description:"copy source" required:"1"`
-	Dest     string `long:"dst" short:"d" description:"copy destination" required:"1"`
-	Preserve string `long:"preserve-props" short:"m" description:"preserve container properties, requires HDF5 library" required:"0"`
+	Source          string `long:"src" short:"s" description:"copy source" required:"1"`
+	Dest            string `long:"dst" short:"d" description:"copy destination" required:"1"`
+	Preserve        string `long:"preserve-props" short:"m" description:"preserve container properties, requires HDF5 library" required:"0"`
+	IgnoreNotsupMod bool   `long:"ignore-notsup-mode-bits" description:"squash unsupported special mode bits when copying to DFS" required:"0"`
```

Comment:
minor: With the assumption that there may be more areas of POSIX noncompliance found, it might make sense to generalize this a little to `IgnoreUnsupported`, with the flag being `--ignore-unsupported`, and the help text being "ignore unsupported filesystem features when copying to DFS". From a maintainability standpoint, I could see this being a little bit of a moving target otherwise. From a UX standpoint, we probably wouldn't want to ask the user to have to figure out which combinations of unsupported things they want to ignore, if we need to add more things that are known not to work but shouldn't be fatal errors.

Reply:
Seems reasonable, I feel like that argument was pretty clunky anyway. This should be more usable.